code/headers/ssd1306_oled_unbuffered.hpp

```diff
@@ -10,44 +9,97 @@ namespace r2d2::display {
      * SSD1306 unbuffered interface for an oled
      * Implements hwlib::window to easily use text and drawing functions that
      * are already implemented. Extends from r2d2::display::ssd1306_i2c_c
+     *
+     * The template parameters are used for the parent class.
      */
-    class ssd1306_oled_unbuffered_c : public ssd1306_i2c_c {
+    template <std::size_t CursorCount, class DisplayScreen>
+    class ssd1306_oled_unbuffered_c
```

Comment:
Honest question: are we actually going to use an unbuffered implementation at all?

Reply:
These needed to exist for testing PO wanted this